### ISSUE TITLE
Expanded data in component inspector

### ISSUE THREAD

**Issue by rightaway** [vuejs/vue-devtools#1192]
### What problem does this feature solve?
It would be nice if props, data and other things in the section under the component inspector were automatically expanded by default. I find I spend a lot of time clicking between things to expand them to get back to where I was.

The first level or two of nesting is expanded, but the ones beyond that aren't. I think it makes more sense to automatically expand everything, and we can collapse the ones we don't need.

Also when I expand something and click to another component and come back to the first one, it's no longer expanded. It should remember the state of what was expanded and what wasn't when switching back and forth between different components.

**Comment by richardeschloss** [comment]
I second that. I was just about to ask for something similar. If I'm on a given component in the inspector, it would be particularly useful if the inspector could stay locked on to the component I'm inspecting (or remember the most recent component) so that if I perform a browser refresh, I can still stay on that component. If I rely on just hot module reloading, I can stay on that component just fine, however, I sometimes find I have to do the browser refresh.

**Comment by rightaway** [comment]
I agree with your idea about staying locked on the component when the browser refreshes.
